Examples/WindowsFormsApplication/readme.md

```diff
@@ -0,0 +1,127 @@
+# Windows Forms sample
```

Comment:
A lot of the questions from above are answered here I think, maybe good to at critical points add some links from the code to specific sections in this doc.

Reply:
I added ms docs link in various places